ui/pages/prelogin/index.js

```diff
@@ -0,0 +1,42 @@
+import LoginLayout from '../../components/layouts/login'
+import Cookies from 'universal-cookie'
+
+export default function PreLogin() {
+  const cookies = new Cookies()
+  const organizations = cookies.get('orgs')
+  console.log(organizations)
```

Comment:
Don't forget to remove this console.log :-)

Reply:
done